migration-from-4x.md

```diff
@@ -40,9 +40,25 @@ In TinyMCE 5.0, some configurations have been removed because they are no longer
 
 #### Removed settings
 
-| **Setting** | **Description** |
-| ----------- | --------------- |
-| `fixed_toolbar_container` | **fixed_toolbar_container**: Owing to the enhancements to the new inline toolbar behaviour, `fixed_toolbar_container` is not required in TinyMCE 5.0. |
+* [`fixed_toolbar_container`](https://www.tiny.cloud/docs/configure/editor-appearance/#fixed_toolbar_container) - Previously, the `fixed_toolbar_container` option was used to render the inline toolbar into a fixed positioned HTML element. This feature has been removed from TinyMCE 5.0 owing to the enhancements to the new inline toolbar behavior.
+
+* **Insert** - Previously, the `insert` option was used to specify the toolbar items to include in the insert toolbar of the **inlite** theme. This feature has been removed from TinyMCE 5.0 owing to the changes in the menus and removal of the `context` property. The `insert` item in the toolbars setting no longer displays a toolbar button by default. The [`insert_button_items`](https://www.tiny.cloud/docs/configure/editor-appearance/#insert_button_items) option is no longer available in TinyMCE 5.0.
```

Comment:
On minor thing here I missed sorry Shikha, the insert button applied to the old UI, not the inlite theme specifically.